packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -90,14 +81,11 @@ export const LabeledInput = React.forwardRef((props, ref) => {
     iconProps,
     inputWrapperProps,
     displayStyle = 'default',
-    iconDisplayStyle = displayStyle === 'default' ? 'block' : 'inline',
     required = false,
     id = uid,
     ...rest
   } = props;
 
-  const icon = svgIcon ?? (status && StatusIconMap[status]());
```

Comment:
is this intentionally removed? the status icons are no longer appearing

Reply:
Added check for message prop with status for icon.

What `null` in `svgIcon` is supposed to be doing?

Reply:
if `null` is passed, default icon should not be rendered

Reply:
Added 👍